config/olatdv.yml

```diff
@@ -4,8 +4,8 @@ idspace: OlatDv
 base_url: /obo/olatdv
 
 products:
-- olatdv.owl: https://raw.githubusercontent.com/obophenotype/developmental-stage-ontologies/master/src/olatdv/olatdv.owl
-- olatdv.obo: https://raw.githubusercontent.com/obophenotype/developmental-stage-ontologies/master/src/olatdv/olatdv.obo
+- olatdv.owl: https://raw.githubusercontent.com/obophenotype/developmental-stage-ontologies/master/src/ontology/components/olatdv.owl
```

Comment:
Is this the historic version frozen in time?

Reply:
Yes, that’s the version currently in the `developmental-stage-ontologies` repo (simply moved to its new location).